### ISSUE TITLE
[Question] Keyboard shortcuts of JavascriptUICommands

### ISSUE THREAD

**Issue by arthursw** [ncsoft/Unreal.js#109]
How can I make the keyboard shortcuts work in _extension-spiralGenerator.js_?

> keyboard shortcut should be routed by some widget which already has keyboard focus.

Could you explain how to route them?

**Comment by arthursw** [comment]
I would like to have global shortcuts (working in different widgets, at least in the `FModeToolkit` and the `FEditorViewportClient`s), is it possible?

**Comment by nakosung** [comment]
I think you might already know you can gain access to input events within FModeToolkit. Global shortcut doesn't seem to be extended easily. For example you cannot add shortcut(ctrl+P) for command palette like sublime, vscode. As I remember it required to modify engine code slightly.
